packages/styled/react/src/styled.tsx

```diff
@@ -120,17 +120,19 @@ function convertUtiltiyToSXFromProps(
 
   if (plugins) {
     for (const pluginName in plugins) {
-      // @ts-ignore
-      [resolvedSxVerbose, , , , sxIgnoreKeys] = plugins[
-        pluginName
-      ]?.inputMiddleWare(
-        resolvedSxVerbose,
-        true,
-        false,
-        Component,
-        componentStyleConfig,
-        ExtendedConfig
-      );
+      if (plugins[pluginName]?.inputMiddleWare) {
```

Comment:
Can you add this condition for `componentMiddleWare` as well?

Reply:
Sure but I think that this condition is already done for componentMiddleWare at lines 2033 and 2050
![image](https://github.com/gluestack/gluestack-ui/assets/47357785/2ffcd558-ad06-45c0-a361-8725eee1eb7d)
![image](https://github.com/gluestack/gluestack-ui/assets/47357785/3c725bb4-3c6c-49ea-91c6-cab2c370326d)

Reply:
Oops, my bad! Will merge the PR.